aspnetcore/mvc/controllers/bind-tryparse/7.0-samples/BindUsingTryParse/BindTryParseMVC/Models/Point.cs

```diff
@@ -0,0 +1,38 @@
+namespace BindTryParseMVC.Models;
+/*
+public class Point : IParseable<Point>
+{
+    public double X { get; set; }
+
+    public double Y { get; set; }
+
+    public override string ToString() => $"({X},{Y})";
+
+    public static Point Parse(string value, IFormatProvider? provider)
+    {
+        if (!TryParse(value, provider, out var result) || result is null)
+        {
+            throw new ArgumentException("Could not parse supplied value.", nameof(value));
+        }
+
+        return result;
+    }
+
+    public static bool TryParse(string? value, IFormatProvider? provider, out Point point)
+    {
+        // Format is "(12.3,10.1)"
+        var trimmedValue = value?.TrimStart('(').TrimEnd(')');
+        var segments = trimmedValue?.Split(',', StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+        if (segments?.Length == 2
+            && double.TryParse(segments[0], out var x)
+            && double.TryParse(segments[1], out var y))
+        {
+            point = new Point { X = x, Y = y };
+            return true;
+        }
+
+        point = new Point();
+        return false;
+    }
+}
```

Comment:
@brunolins16  wrote:
> I am not sure if it is in the docs yet, but do you think we need to mention about `IParseable<T>`

@fiyazbinhasan  suggestion, I added @DamianEdwards sample here. But when I run the app I get
```
Unhandled exception. System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load type 'System.IParseable`1' from assembly 'System.Runtime, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
```
It doesn't like like `IParseable<T>` is ready.

Reply:
I agree.

Reply:
There's no "e" in it now: `IParsable`